client/src/ui/molecules/document-survey/surveys.ts

```diff
@@ -93,11 +95,23 @@ export const SURVEYS: Survey[] = [
   {
     key: SurveyKey.HOMEPAGE_FEEDBACK_2024,
     bucket: SurveyBucket.HOMEPAGE_FEEDBACK_2024,
-    show: (doc: Pick<Doc, "mdn_url">) => /[^/]+\/$/i.test(doc.mdn_url),
+    show: (doc: Pick<Doc, "mdn_url">) => /^\/[^/]+\/$/i.test(doc.mdn_url),
     src: "https://survey.alchemer.com/s3/8075407/MDN-Homepage-Improvements",
     teaser: "We are refreshing our homepage and would love",
     question: "your input",
     ...survey_duration(SurveyBucket.HOMEPAGE_FEEDBACK_2024),
     ...survey_rates(SurveyKey.HOMEPAGE_FEEDBACK_2024),
   },
+  {
+    key: SurveyKey.WEBDX_EDITING_2024,
+    bucket: SurveyBucket.WEBDX_EDITING_2024,
+    show: (doc: Pick<Doc, "mdn_url">) =>
+      /^\/[^/]+\/docs\/Web\/API($|\/.*$)/i.test(doc.mdn_url),
```

Comment:
Please exclude `/de/` from this, to avoid two surveys from showing up simultaneously (unless we already prevent this, and show only the first matching one?).

Reply:
That's what it does, taking the first one only.